Demo/Demo/SwiftUIComponents/PayPalWebViews/PayPalOrderActionButton.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+struct PayPalOrderActionButton: View {
+
+    let intent: Intent
+    let orderID: String
+    let selectedMerchantIntegration: MerchantIntegration
+
+    @ObservedObject var paypalWebViewModel: PayPalWebViewModel
+
+    var body: some View {
+        ZStack {
+            Button("\(intent.rawValue)") {
+                completeOrder()
+            }
+            .buttonStyle(RoundedBlueButtonStyle())
+            .padding()
+
+            if .loading == paypalWebViewModel.state.authorizedOrderResponse ||
+                .loading == paypalWebViewModel.state.capturedOrderResponse {
+                CircularProgressView()
+            }
+        }
+    }
+
+    private func completeOrder() {
+        if intent == .capture {
```

Comment:
Since intent is an enum can we use a switch here instead?

Reply:
I just used if since there are only two cases.